sql/core/src/test/resources/sql-tests/results/pgSQL/boolean.sql.out

```diff
@@ -0,0 +1,708 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 79
+
+
+-- !query 0
+SELECT 1 AS one
+-- !query 0 schema
+struct<one:int>
+-- !query 0 output
+1
+
+
+-- !query 1
+SELECT true AS true
+-- !query 1 schema
+struct<true:boolean>
+-- !query 1 output
+true
+
+
+-- !query 2
+SELECT false AS false
+-- !query 2 schema
+struct<false:boolean>
+-- !query 2 output
+false
+
+
+-- !query 3
+SELECT boolean('t') AS true
+-- !query 3 schema
+struct<true:boolean>
+-- !query 3 output
+true
+
+
+-- !query 4
+SELECT boolean('true') AS true
+-- !query 4 schema
+struct<true:boolean>
+-- !query 4 output
+true
+
+
+-- !query 5
+SELECT boolean('test') AS error
+-- !query 5 schema
+struct<error:boolean>
+-- !query 5 output
+NULL
+
+
+-- !query 6
+SELECT boolean('false') AS false
+-- !query 6 schema
+struct<false:boolean>
+-- !query 6 output
+false
+
+
+-- !query 7
+SELECT boolean('foo') AS error
+-- !query 7 schema
+struct<error:boolean>
+-- !query 7 output
+NULL
+
+
+-- !query 8
+SELECT boolean('y') AS true
+-- !query 8 schema
+struct<true:boolean>
+-- !query 8 output
+true
+
+
+-- !query 9
+SELECT boolean('yes') AS true
+-- !query 9 schema
+struct<true:boolean>
+-- !query 9 output
+true
+
+
+-- !query 10
+SELECT boolean('yeah') AS error
+-- !query 10 schema
+struct<error:boolean>
+-- !query 10 output
+NULL
```

Comment:
Nit: we need to open a JIRA in https://issues.apache.org/jira/browse/SPARK-27764 and list the behavior difference. Add the comment and explain the difference in the boolean.sql file with the JIRA number.

Reply:
Could you go over the remaining queries and compare them with https://github.com/postgres/postgres/blob/REL_12_BETA1/src/test/regress/expected/boolean.out?

Reply:
Done